fittotcx/program.py

```diff
@@ -234,12 +234,17 @@ def main():
             description="This program takes a FIT file and converts it " +
                         "into an TCX file and output the result to the " +
                         "standard output.")
-    parser.add_argument('file', metavar='FILE', type=argparse.FileType('r'))
+    parser.add_argument('inputfile', metavar='INPUTFILE', nargs=1, type=str)
+    parser.add_argument('outputfile', metavar='OUTPUTFILE', nargs='?', type=str, default=None)
     args = parser.parse_args()
 
     try:
-        document = convert(sys.argv[1])
-        sys.stdout.write(documenttostring(document).decode('utf-8'))
```

Comment:
If you leave `args.inputfile` and `args.outputfile` as objects, all that's needed here is:

```diff
-        document = convert(sys.argv[1])
-        sys.stdout.write(documenttostring(document).decode('utf-8'))
+        document = convert(args.inputfile)
+        args.outputfile.write(documenttostring(document).decode('utf-8'))
```